### PR TITLE
Fix status bar color in MainActivity

### PR DESCRIPTION
**Pull request by Domi04151309**
Very simple change

| Before | After |
| --- | --- |
| before | after |

### CODE REVIEW

File: app/src/main/java/com/sapuseven/untis/activities/MainActivity.kt [hunk 1]
```diff
@@ -10,9 +10,7 @@ import android.net.Uri
 import android.os.Bundle
 import android.os.Handler
 import android.os.Looper
-import android.view.MenuItem
-import android.view.View
-import android.view.ViewGroup
+import android.view.*
```
Comment (SapuSeven) [inline app/src/main/java/com/sapuseven/untis/activities/MainActivity.kt:13]: Any reason for that wildcard import?

Comment (Domi04151309) [reply app/src/main/java/com/sapuseven/untis/activities/MainActivity.kt:13]: It was automatically created by Android Studio

Comment (SapuSeven) [reply app/src/main/java/com/sapuseven/untis/activities/MainActivity.kt:13]: Alright, in this case it's acceptable.

**Review by SapuSeven — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/sapuseven/untis/activities/BaseActivity.kt [hunk 2]
```diff
@@ -18,7 +18,7 @@ import java.io.File
 
 @SuppressLint("Registered") // This activity is not intended to be used directly
 open class BaseActivity : AppCompatActivity() {
-	private var currentTheme: String = ""
+	var currentTheme: String = ""
```
[the remark this answers is not in the document]
Comment (Domi04151309) [reply app/src/main/java/com/sapuseven/untis/activities/BaseActivity.kt]: Yes that makes sense

**Review by SapuSeven — APPROVED** [review]

**Pull request merged**